### ISSUE TITLE
Update the "CODE" block in RStudio Connect HTML previews

### ISSUE THREAD

**Issue by sellorm** [rstudio/pins-python#114]
This relates to [similar ticket in the R pins repo](https://github.com/rstudio/pins/issues/615).

The code block in the HTML previews used when publishing to RStudio Connect is currently empty.

This is a useful tool for new-user on-boarding and should be pre-populated with the information required to get started using this pin.

In the above ticket, I proposed that this section is updated along the lines of the below (though the examples offered by pins-for-R and pins-for-python should always match where possible.  These examples should have the appropriate URLs and pin names substituted in):

---

## Code

The following examples assume you have your Connect API key stored in a `CONNECT_API_KEY` environment variable. Please refer to the documentation for more information and information on other supported authentication modes.

### R

``` R
library(pins)
board <- board_rsconnect("envvar", server = "https://connect.example.com")
pin_read(board, "username/pin_name")
```

[Pins for R documentation](https://pins.rstudio.com/reference/board_rsconnect.html)

### Python

``` python
import pins
board = pins.board_rsconnect(server_url = "https://connect.example.com")
board.pin_read("username/pin_name")
```

[Pins for python documentation](https://rstudio.github.io/pins-python/api/api_card/pins.board_rsconnect.html#pins.board_rsconnect)

---

This could perhaps be collapsible, though it should be clear even when collapsed that this section contains R and Python code.

More than anything this is about on-boarding new users to the pins ecosystem more quickly if someone within their organisation has shared a pin with them.

**Comment by machow** [comment]
Opened https://github.com/rstudio/pins-python/pull/142 to put in a code block that matches the R one.

One unique challenge in python pins is that how we represent the board, depends on how a user instantiated. For example, if a person created a board using...

```python
board_rsconnect(..., allow_pickle_read=True)
```

Then we need to put `allow_pickle_read=True` on the board. I could see this getting weird. For example, if an organization did not want people reading pickle files.

**Comment by isabelizimm** [comment]
addressed in #223
